include/lapi/rt_sigaction.h

```diff
@@ -190,8 +198,9 @@ static int ltp_rt_sigaction(int signum, const struct sigaction *act,
 		kact.k_sa_handler = act->sa_handler;
 		memcpy(&kact.sa_mask, &act->sa_mask, sizeof(sigset_t));
 		kact.sa_flags = act->sa_flags;
+#ifdef HAVE_SA_RESTORER
 		kact.sa_restorer = NULL;
-
+#endif
```

Comment:
Yes, you are right. Should I make a new pull request?

Huacai

On Tue, Aug 30, 2016 at 9:00 PM, Cyril Hrubis <notifications@github.com>
wrote:

> In include/lapi/rt_sigaction.h
> <https://github.com/linux-test-project/ltp/pull/85#discussion_r76789136>:
>
> >  		kact.sa_restorer = NULL;
> > -
> > +#endif
>
> I'm not sure about this part, shouldn't we ifdef this initialization only
> on #ifdef __mips__ since the kernel may check that field is NULL if it's
> not used.
>
> Otherwise the patch looks fine.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/linux-test-project/ltp/pull/85/files/dae2b7fb04156c32abfcfd7561a58a46cd3c3b7d#r76789136>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AURiMmy1oenpl19yxL1IEo83EAl04_kbks5qlCldgaJpZM4JvFWw>
> .
>